src/System.Transactions.Local/src/System/Transactions/Transaction.cs

```diff
@@ -319,9 +317,9 @@ public override int GetHashCode()
 
         // Don't allow equals to get the identifier
         //
-        public override bool Equals(object obj)
+        public override bool Equals(object? obj)
         {
-            Transaction transaction = obj as Transaction;
```

Comment:
Nit: this method method could be simplified to a single return expression if using the `is` operator.